src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/forgotten_password.yaml

```
@@ -0,0 +1,25 @@
+twig_hooks:
+    hooks:
+        'sylius_shop.account.forgotten_password':
```

I would be for old name 🤔 
```suggestion
        'sylius_shop.account.reset_password':
```
here, and everywhere else

---

We already have template called reset_password, so in this case we should stay with this name